cypress/plugins/buildProject.js

```diff
@@ -1,13 +1,14 @@
 const { join } = require("path");
-const { spawnSync } = require("child_process");
+const execa = require("execa");
 
 // Build the given NextJS project
 const buildProject = ({ project }, config) => {
   process.stdout.write(`Building project: ${project}...`);
 
   // Build project
-  spawnSync("npm", ["run", "build"], {
+  execa.sync("npm", ["run", "build"], {
     cwd: join(config.buildsFolder, project),
+    preferLocal: true,
```

Comment:
See https://github.com/sindresorhus/execa#preferlocal
This removes the need for `../node_modules/.bin/...`.